code/modules/recycling/sortingmachinery.dm

```diff
@@ -185,6 +188,15 @@
 			wrapped_item.AddComponent(/datum/component/pricetag, sticker.payments_acc, sticker.cut_multiplier)
 		update_appearance()
 
+	if(istype(item, /obj/item/boxcutter))
+		if( (item.icon_state == "boxcutter_on"))
+			unwrap_contents(user)
```

Comment:
can you make this use the full unwrap code instead of shortcut to the end? It still takes time to open packages with a boxcutter